### PR TITLE
Update helpdesk page #153

### PR DESCRIPTION
**Pull request by gruunday**

### CODE REVIEW

File: source/help/index.md [hunk 1]
```diff
@@ -5,13 +5,13 @@ type: cmt
 banner: helpdesk.jpg
 cmt:
   - position: Helpdesk
-    name: Tom Doyle
-    nick: greenday
-    image: greenday.jpg
+    name: Sahil Mehra
+    nick: pr0n
+    image: pr0n.jpg
```
Comment (butlerx) [inline source/help/index.md]: Images need to be added

File: source/help/index.md [hunk 2]
```diff
@@ -5,13 +5,13 @@ type: cmt
...
   - position: Helpdesk
-    name: Dylan McDonald
-    nick: grace
-    image: grace.jpg
+    name: Michal Durinik
+    nick: mikello
+    image: mikello.jpg
```
Comment (butlerx) [inline source/help/index.md]: Smae as above

File: source/help/index.md (hunk 1 above)
Comment (gruunday) [inline source/help/index.md]: Yeah but when it is it should follow that name. So this can sit here until the other Images are added?

Comment (butlerx) [inline source/help/index.md]: 5his will probably create an error on the page 
Have you tried generating

**Pull request merged**